test/functional/verify/controls/azure_network_security_group.rb

```diff
@@ -0,0 +1,24 @@
+resource_group = attribute('resource_group',            default: nil)
```

Comment:
For consistency, we may eventually want to reclass these tests as "integration" tests (that's what the inspec project calls any test that does live-fire tests against a specific platform target).  inspec project uses the word 'functional' to refer specifically to UX testing.  Not a blocker issue.

Reply:
We can rename the folder that's not a big deal.